test/mocha/30-nonce.js

```diff
@@ -124,5 +124,49 @@ describe('Nonce API', () => {
       err.message.should.equal('Invalid "entryStyle" "not-machine-or-human"; ' +
         '"entryStyle" must be "human" or "machine".');
     });
+    it('should generate a new nonce if one already exists', async () => {
+      const accountId = mockData.accounts['alpha@example.com'].account.id;
+      const actor = await brAccount.getCapabilities({id: accountId});
+      let err;
+      let result1;
+      let result2;
+      // create a new nonce token
+      await brAuthnToken.set({
+        account: accountId,
+        actor,
+        type: 'nonce',
+      });
+      try {
+        result1 = await brAuthnToken.get({
+          account: accountId,
+          actor,
+          type: 'nonce',
+        });
+      } catch(e) {
+        err = e;
+      }
+      assertNoError(err);
+      should.exist(result1);
+
+      // create another nonce token
+      await brAuthnToken.set({
+        account: accountId,
+        actor,
+        type: 'nonce',
+      });
+
+      try {
+        result2 = await brAuthnToken.get({
+          account: accountId,
+          actor,
+          type: 'nonce',
+        });
+      } catch(e) {
+        err = e;
+      }
+      assertNoError(err);
+      should.exist(result1);
+      result1.should.not.deep.equal(result2);
```

Comment:
This is good, but instead/in addition, please make more explicit assertions about the `nonce` not being the same as the earlier one.  Also, `eql` is a shortcut for `deep.equal`.